src/utils/mod.rs

```diff
@@ -74,6 +77,56 @@ pub fn read_key_file<P: AsRef<Path>>(path: P) -> anyhow::Result<InMemorySigner>
     }
 }
 
+pub fn read_ledger_keypair() -> anyhow::Result<InMemorySigner> {
+    //TODO: use prompt to get the HD derivation path from the user.
+    let input_seed_phrase_path = BIP32Path::from_str("44'/397'/0'/0'/1'").unwrap();
```

Comment:
Should `"44'/397'/0'/0'/1'"` be provided like a command line argument?

Reply:
This method will be refactored as part of the generic signer struct.